strings/base_coroutine_threadpool.h

```diff
@@ -434,6 +447,21 @@ WINRT_EXPORT namespace winrt
                 m_handle(handle)
             {}
 
+#if defined(__GNUC__) && !defined(__clang__)
+            // HACK: GCC seems to require a move when calling operator co_await
+            // on the return value of resume_on_signal.
+            // This might be related to upstream bug:
+            // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=99575
+            awaitable(awaitable &&other) noexcept :
```

Comment:
Can't we use `= default` to force the compiler to generate this?

Reply:
IIRC the issue is that `std::atomic<T>` is unmovable (copy constructor is deleted).

Reply:
Ah, that's too bad.